### PR TITLE
Risk Assessment scraping and table

### PR DESCRIPTION
**Pull request by thebioengineer**
addresses #112
- function for scraping risk assessments
- kable-ing function

### CODE REVIEW

File: R/risk_table.R [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#' @rdname scrape
+#'
+vt_kable_risk_assessment <- function(x, format = NULL){
+  t <- kable(x,
+        format = format,
+        longtable =  TRUE,
+        col.names = c("Requirement Name", "Requirement ID", "Risk Assessment") )
+  t <- column_spec(t, 1, border_right = TRUE)
+  t <- collapse_rows(t, c(1))
+  t
```
Comment (mariev) [inline R/risk_table.R:57]:
```suggestion
  t <- kable_styling(t, latex_options = "hold_position")
  t
```

**Review by mariev — COMMENTED** [review]
should we have a test case for `vt_kable_risk_assessment`? at the very least verify that the kableExtra object is being generated?

blocked by #116 b/c need to add this to template usage

File: R/risk_table.R [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+#' @param x data.frame as output from \code{\link{vt_scrape_risk_assessment}}
+#' @param format passed to \code{kable}
+#' @return knitr_kable object
+#' @importFrom knitr kable
+#' @importFrom kableExtra collapse_rows column_spec
+#' @export
+#'
+#' @rdname scrape
+#'
+vt_kable_risk_assessment <- function(x, format = NULL){
```
Comment (thebioengineer) [inline R/risk_table.R:50]: Neet to add `vt_renter_to()`

**Review by mariev — APPROVED** [review]
made some updates, not sure who is reviewing whom here, but looks good 👍

**Pull request merged**